src/Engine/Evt/CMakeLists.txt

```diff
@@ -1,19 +1,19 @@
 cmake_minimum_required(VERSION 3.27 FATAL_ERROR)
 
 set(ENGINE_EVENTS_SOURCES
-        EventEnums.cpp
-        EventIR.cpp
-        EventMap.cpp
-        EventInterpreter.cpp
+        EvtEnums.cpp
+        EvtInstruction.cpp
+        EvtProgram.cpp
+        EvtInterpreter.cpp
         Processor.cpp)
```

Comment:
no EvtProcessor?

Reply:
Decided I'll need to look a bit deeper into the code in Processor before renaming X).